index.html

```diff
@@ -0,0 +1,129 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <title>Coding Challenges</title>
+</head>
+<body>
+  <div class="htmlbody">
+  <div class="bgImg">
+    <nav class="navbar fixed-top topNav">
+      <nav class="heading">
+        <img class="heading-elements" src="images/facebook.jpg" width="24" height="24" alt="facebook">
+        <img class="heading-elements" src="images/twitter.png" width="24" height="24" alt="twitter">
+        <span class="text-white fw-bold heading-elements">English</span>
+        <span class="text-white fw-bold heading-elements">My Page</span>
+        <span class="text-white fw-bold heading-elements">Logout</span>
+      </nav>
+      <div class="container bottomNav">
+        <!-- Brand Logo -->
+        <a class="nav-desktop navbar-brand" href="#"><img src="images/cc-logo.jpg" width="48" alt="logo"></a>
+        <!-- Contents -->
+        <a class="nav-desktop text-decoration-none" id="content-cc" href="#contents">Contents</a>
+        <!-- Coding Challenges -->
+        <a class="nav-desktop text-decoration-none" id="challenge-cc" href="#coding-challenges">Coding Challenges</a>
+        <!-- About -->
+        <a class="nav-desktop text-decoration-none" id="about-cc" href="about.html">About CC</a>
+        <!-- Hamburger menu -->
+        <img id="hamburger" src="images/bars-solid.jpg" width="36" height="48" alt="menu-icon">
+      </div>
+    </nav>
+    
+    <!-- Navigation Menu -->
+    <div class="close-menu" id="navMenu">
+        <img id="close" src="images/x-solid.svg" width="24"  alt="close">
+        <ul id="nav-list" class="text-decoration-none list-unstyled ms-auto">
+            <li class="mx-auto">
+                <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="mx-auto">
+                <a class="nav-link" href="about.html">About</a>
+            </li>
+        </ul>
+    </div>
+
+    <section class="container-fluid home-page">
+        <h3 class="motto">"Preparing you for the language of the future!"</h3>
+        <h1 class="description-header">CODING CHALLENGES</h1>
+    </section>
+    <div id="describer">
+        <p id="description">Wanna challenge yourself with interesting programming tasks? Get started now by proceeding to the 'Coding Challenges' section of this page.</p>
+    </div>
+    <div>
+      <h2 class="date cocogoose">2015.10.15(THU) ~ 16(FRI)</h2>
+      <p class="loc lato">@National Museum of Korea. Art Center Nabi and more</p>
+    </div>
+  </div>
```

Comment:
### Your Date
![wanna-challenge (copy)](https://user-images.githubusercontent.com/93039359/181396015-89512f6c-fb40-45d6-a0ca-834967c8bcba.png)

### Design Date
![design-date](https://user-images.githubusercontent.com/93039359/181396058-a1131133-9340-47ac-a7f1-794272311fa4.png)


- [ ] Your date is not visible and it should not be inside the work section. Please fix this issue :blush: